README.md

```diff
@@ -1,26 +1,28 @@
 # GitHub OAuth Ruby Quickstart
+
+[DEGW: An intro here motivating the problem you are trying to help solve. Explain (briefly) what an OAuth app is, and why you might want to build one. Why do we need a guide? Is it hard to do? (Narrator: In fact, it is hard.) ALso give a preview of what the reader can expect to have achieved by the end of the article.]
+
+[DEGW: More thoughts. I am certainly guilty of this myself, but I am finding I dislike the word "user" to describe people who will be using your application. We should think about ways to avoid this word in our writing. It will not be easy.]
+
 ## About this Quickstart Guide
 This is a quickstart guide to GitHub OAuth authentication using:
 - Ruby
-- A simple web framework called [Sinatra](http://sinatrarb.com)
+- The [Sinatra](http://sinatrarb.com) web framework
 - [Octokit](https://github.com/octokit/octokit.rb) - a Ruby client for the GitHub [REST API](https://github.com.) (makes interacting with GitHub data easier)
 
 To demonstrate the OAuth process, we're going to:
-1) Set up a simple web server with Sinatra
+1) Set up a basic web server with Sinatra
 2) Register a new OAuth Application on GitHub.com
 3) Authenticate a user into our Sinatra app using GitHub's OAuth API
 4) Use Octokit to retrieve this user's information from GitHub
 
-### Prerequisites
-If you haven't signed up for the [GitHub Developer Program](https://developer.github.com/program) yet, you'll need to do that before starting this guide.
-
```

Comment:
@DEGoodmanWilson I actually thought it was necessary to sign up for the developer program in order to register a new app. I agree this is unnecessary given the previous is not true.